cli/src/cmd/exec.rs

```diff
@@ -430,10 +430,8 @@ impl Cli {
                     PathOrUrl::Path(v) => v.to_str().unwrap_or_else(|| ""),
                     PathOrUrl::Url(v) => v.as_str(),
                 };
-                let name = match name {
-                    Some(v) => v,
-                    None => "",
-                };
+                let default_name = "".to_string();
+                let name = name.unwrap_or_else(|| &default_name);
```

Comment:
```suggestion
let name = name.unwrap_or_default();
```

I believe `unwrap_or_default` should do this for you. Just curious if this relies on database/server validation for an empty string name, or if that is valid?

Reply:
`unwrap_or_default()` didn't work because the type is a `&String`, which doesn't implement `Default`.  I also couldn't turn that into a one liner with `unwrap_or_else` because I can't create a reference in the closure and have that returned. This seemed like the most terse way to accomplish this but I def could be missing something

Reply:
Ah I see -- ok, just pushed up a small tweak.

Reply:
Thanks, good to know of that technique!